usr.sbin/service/service.8

```diff
@@ -161,10 +195,11 @@ _service () {
 complete -F _service service
 .Ed
 .Sh SEE ALSO
-.Xr bash 1 Pq Pa ports/shells/bash ,
 .Xr rc.conf 5 ,
 .Xr rc 8 ,
-.Xr rcorder 8
+.Xr sysrc 8 ,
+.Xr rcorder 8 ,
+.Xr bash 1 Pq Pa ports/shells/bash .
```

Comment:
why is this at the end now?

Reply:
because shells/bash isn't even in the base system, so it's the least useful Xr in the manpage and should be listed last.  to be honest, i'm not even sure why it's listed at all; if it's because of the completion example, should `csh(1)` be listed too?

Reply:
i dunno, i would remove ax it.

Reply:
ok, removed!